models/groupusers.js

```diff
@@ -17,7 +17,7 @@ module.exports = (sequelize, DataTypes) => {
   };
   GroupUsers.init({
     id: DataTypes.INTEGER,
-    userIds: DataTypes.INTEGER,
+    userIds: DataTypes.ARRAY,
```

Comment:
Is this correct? I think that INTEGER still should be here
Also, you should change this key from "userIds" to "userId"